constants/queryKeys.ts

```diff
@@ -1,25 +1,24 @@
-import { NetworkId } from '@synthetixio/contracts-interface';
 import { WeiSource } from '@synthetixio/wei';
 import { SPACE_KEY } from 'constants/snapshot';
 import { Synths } from './currency';
 
 export const QUERY_KEYS = {
 	Debt: {
-		WalletDebtData: (walletAddress: string, networkId: NetworkId) => [
+		WalletDebtData: (walletAddress: string, networkId: number) => [
```

Comment:
Do we need to initialise the query context if the network is not defined?

Reply:
hmm not totally sure I understand the question. This file (which I dont think I've ever used before) is for defining cache keys for `useQuery`, quite smart to have that in one place. Then you can invalidate and refetch specific queries without having access to `refetch`. So instead of passing around a `refetch` function we could `queryClient.invalidateQueries(QUERY_KEYS.Debt.WalletDebtData(address, networkId))`

Reply:
Yes I think you are right for this file specifically. Good to know (similar to apollo).